sourcegraph/sourcegraph
# bext: fix GitHub client-side routing support

Closes #[31716](https://github.com/sourcegraph/sourcegraph/issues/31716)

Adds client-side routing support to the code navigation view. 

## Test plan

- `sg run bext`
- open any repository and navigate through files without reloading the page
- ensure code intel works as expected in all files
